pkg/kubeapiserver/authenticator/config.go

```diff
@@ -30,15 +30,15 @@ import (
 	"k8s.io/apiserver/pkg/authentication/request/x509"
 	"k8s.io/apiserver/pkg/authentication/token/tokenfile"
 	"k8s.io/apiserver/pkg/authentication/user"
+	"k8s.io/apiserver/plugin/pkg/authenticator/password/keystone"
```

Comment:
I wasn't expecting all these to move.  We have an apiserver dependency or is it just kubeapiserver?